docs/guide/migration.rst

```diff
@@ -9,4 +9,193 @@ This is a guide to migrate from Stable-Baselines to Stable-Baselines3.
 
 It also references the main changes.
 
-**TODO**
+.. warning::
+	This section is still a Work In Progress (WIP) Things might be added in the future before 1.0 release.
```

Comment:
I think this part could be removed as we are near the release and probably won't update this doc much after release?